app/components/TripCard.tsx

```diff
@@ -1,10 +1,87 @@
-import { View, Text } from "react-native"
+import { View, ImageBackground, ViewStyle, StyleProp } from "react-native"
 import React from "react"
+import { Card } from "./Card"
+import { Icon } from "./Icon"
+import { colors, typography } from "../theme"
+import Divider from "./Divider"
+import { Text } from "./Text"
```

Comment:
I beleive you can also run ``yarn lint`` to check these ^^. But I can make a pre-commit hook so it shows up there